serializers/postgres.rb

```diff
@@ -13,7 +13,8 @@ def self.serialize_internal(pg, options = {})
       ha_type: pg.ha_type,
       flavor: pg.flavor,
       ca_certificates: pg.ca_certificates,
-      maintenance_window_start_at: pg.maintenance_window_start_at
+      maintenance_window_start_at: pg.maintenance_window_start_at,
+      read_replica: pg.read_replica? || false
```

Comment:
Do we need `|| false` part?

Reply:
If we want to convert `nil` to `false`, `!!pg.read_replica?` does it